commons_ocollection/ograph_extended.ml

```diff
@@ -301,19 +317,25 @@ let launch_gv_cmd filename =
   *)
   ()
 
-let print_ograph_extended g filename launchgv =
+let display_graph_cmd =
+  match get_os with
+  | `MacOs -> launch_png_cmd
+  | `Linux -> launch_gv_cmd
```

Comment:
Is there a reason why .ps is better than .png for this purpose? Could we use .png for both? Or .pdf?

Reply:
No reason. I was just familiar with postscript and gv when I wrote this code a long time ago.